fbchat/graphql.py

```diff
@@ -127,6 +128,64 @@ def graphql_to_attachment(a):
         return Attachment(
             uid=a.get('legacy_attachment_id')
         )
+    
+def graphql_to_poll(a):
+    rtn = Poll(
+        title=a.get('title') if a.get('title') else a.get("text"),
+        options=[graphql_to_poll_option(m) for m in a.get('options')]
+    )
+    rtn.uid = a.get("id")
+    rtn.options_count = a.get("total_count")
+    return rtn
+
+def graphql_to_poll_option(a):
+    rtn = PollOption(
+        text=a.get('text'),
+        vote=a.get('viewer_has_voted') == 'true' if isinstance(a.get('viewer_has_voted'), str) else a.get('viewer_has_voted')
+    )
+    rtn.uid = a.get('id')
+    rtn.voters = [m.get('node').get('id') for m in a.get('voters').get('edges')] if isinstance(a.get('voters'), dict) else a.get('voters')
+    rtn.votes_count = a.get('voters').get('count') if isinstance(a.get('voters'), dict) else a.get('total_count')
+    return rtn
+
+def graphql_to_event(a):
```

Comment:
Sometimes, `oid` is set as the event ID, see the comment on `client.py/fetchEventInfo`

Reply:
Done, I didn't see that 😃